bin/ds

```diff
@@ -162,9 +180,6 @@ function build {
     source ~/.bashrc
   fi
 
-  echo 'Adding APC script at /apc.php...'
-  ln -s /srv/salt/php5/apc.php $WEB_PATH/apc.php
-
```

Comment:
I don't provide this file in new Vagrant boxes. We don't use APC cache anymore, it has been replaced by Zend OPcache.